mungegithub/mungers/sync/issue-sync.go

```diff
@@ -50,7 +50,7 @@ type Priority int
 
 // String return the priority label in string
 func (p Priority) String() string {
-	return fmt.Sprintf(priorityPrefix+"%d", p)
+	return "priority/failing-test"
```

Comment:
this will work, but I"m not sure it's the best place to do it.  I think we should create a new const PriorityLabel called PriorityFailingTest = Priority("failing-test") // the string func still needs update